src/endpoints/accounts/entities/account.verification.ts

```diff
@@ -7,15 +7,15 @@ export class AccountVerification {
     Object.assign(this, init);
   }
 
-  @ApiProperty({ description: 'Source code hash' })
+  @ApiProperty({ description: 'Source code hash' , required: false })
```

Comment:
this entity is used only for account verification endpoint
codeHash should be required